source/OptimaJet.DWKit.StarterApplication/Forms/Groups/UpdateForm.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Linq;
+using JsonApiDotNetCore.Internal;
+using Microsoft.EntityFrameworkCore;
+using Optimajet.DWKit.StarterApplication.Models;
+using OptimaJet.DWKit.StarterApplication.Repositories;
+using OptimaJet.DWKit.StarterApplication.Services;
+
+namespace OptimaJet.DWKit.StarterApplication.Forms.Groups
+{
+    public class UpdateForm : BaseForm
+    {
+        public UserRepository UserRepository { get; set; }
+        public GroupRepository GroupRepository { get; set; }
+        public ICurrentUserContext CurrentUserContext { get; }
+        public UpdateForm(
+            UserRepository userRepository,
+            GroupRepository groupRepository,
+            ICurrentUserContext currentUserContext)
+        {
+            UserRepository = userRepository;
+            GroupRepository = groupRepository;
+            CurrentUserContext = currentUserContext;
+         }
+        public bool IsValid(int id, Group group)
+        {
+            var retVal = true;
+            //If changing organization, validate the change
+            if (group.OwnerId > 0)
+            {
+                var currentUser = UserRepository.GetByAuth0Id(CurrentUserContext.Auth0Id).Result;
+                var orgIds = currentUser.OrganizationIds ?? Enumerable.Empty<int>();
+                if (!orgIds.Contains(group.OwnerId))
+                {
+                    retVal = false;
+                    var original = GroupRepository.Get()
+                                                  .Where(g => g.Id == id)
+                                                  .Include(g => g.Owner)
+                                                  .FirstOrDefaultAsync().Result;
+                    var message = $"Project '{original.Name}': Current user not a member of project organization '{original.Owner.Name}'";
+                    var error = new Error(422, "Project Validation Failure", message);
```

Comment:
The error says "Project Validation failure", but this is a "Group" Update form.

Reply:
A couple of problems within the message and error referring to project when this is group.  These have been updated.

Reply:
has it been pushed?